homeassistant/components/sensor/gtfs.py

```diff
@@ -219,11 +223,16 @@ def icon(self):
         """Icon to use in the frontend, if any."""
         return ICON
 
+    @property
```

Comment:
Having it in the attributes might be useful for some notifications or that kind of things. I'll fix this, thanks for the pointers!